index.bs

```diff
@@ -8560,13 +8577,33 @@ it indicates that the construct is exposed
 on that particular set of global interfaces.
 
 The [{{Exposed}}] [=extended attribute=] must either
-[=takes an identifier|take an identifier=] or
-[=takes an identifier list|take an identifier list=].
+[=takes an identifier|take an identifier=],
+[=takes an identifier list|take an identifier list=] or
+[=takes a wildcard|take a wildcard=].
 Each of the identifiers mentioned must be a [=global name=] and be unique.
-This list of identifiers is known as the construct's
-<dfn>own exposure set</dfn>.
+
+The <dfn>own exposure set</dfn> is the set of [=global name=]s which
+[{{Exposed}}] is applied to.
+
+If the [{{Exposed}}] [=extended attribute=]
+[=takes an identifier|take an identifier=] or
+[=takes an identifier list|take an identifier list=],
+the [=own exposure set=] is the list of identifiers.
+If the [{{Exposed}}] [=extended attribute=]
+[=takes a wildcard|take a wildcard=], the [=own exposure set=] is the
+the set of interfaces which have the [{{Global}}] extended attribute.
+
+<p class="advisement">
+    <code>[Exposed=*]</code> should be used with care. It is only appropriate
+    when an interface does not expose significant new capabilities. If the
+    interface may be restricted or disabled in some environments, it is
+    recommended to list the globals explicitly.
+</p>
 
 <div algorithm>
+*   If the [{{Exposed}}] [=extended attribute=] [=takes a wildcard=], then the
+    [=exposure set=] is the set of all interfaces which have a [{{Global}}]
+    or [{{PrimaryGlobal}}] [=extended attribute=].
```

Comment:
Is this bit necessary? Isn't that all handled by the below algorithm already?

Reply:
Sorry, I just had this copied in there for my own use when writing the patch and forgot to remove it. Removed now.